api/schemas.py

```diff
@@ -15,6 +15,8 @@ def validate_string(value, *, choices=None):
 
 
 OBSERVABLE_TYPE_CHOICES = (
+    's1_agent_id',
+    'ms_machine_id',
```

Comment:
It looks like all of these types were listed alphabetically before. Maybe we should keep this order?